### DISCUSSION TITLE
I have not yet received the benefits of the GitHub student Developer Pack.

### DISCUSSION THREAD

**Discussion by mann-rana29** [community/community#134497] [GitHub Education]
### Hi everyone,

its been 5 days and i still havent received any mail from github

**Comment by hubertit** [comment]
Hi there,

The approval process for the GitHub Student Developer Pack can take up to a few weeks. If it's been 5 days, it's best to wait a bit longer. Make sure you’ve provided all necessary documentation and that your email is correct. If you still don't receive it after a couple of weeks, you might want to contact GitHub support for further assistance.

Good luck!

**Comment by prabhjotschugh** [comment]
Hi @the25thjin  ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks;

When the benefits become available, the status label will turn purple and change to:

`Congrats! Your request for @the25thjin was approved and your benefits are now available.`

For further reference you can look at these discussions: [#132790](https://github.com/orgs/community/discussions/132790) , [#131665](https://github.com/orgs/community/discussions/131665), [#132470](https://github.com/orgs/community/discussions/132470) , [#133531](https://github.com/orgs/community/discussions/133531), [#134279](https://github.com/orgs/community/discussions/134279)

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

[How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
[How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
